app/controllers/api/v2/hostgroups_controller.rb

```diff
@@ -34,6 +34,7 @@ def show
           param :compute_profile_id, :number, :desc => N_('Compute profile ID')
           param :operatingsystem_id, :number, :desc => N_('Operating system ID')
           param :architecture_id, :number, :desc => N_('Architecture ID')
+          param :pxe_loader, Operatingsystem.all_loaders, :desc => N_("DHCP filename option (Grub2/PXELinux by default)"), :required => true
```

Comment:
I think the flag is only required for PXE installations. `operatingsystem_id` does not have a required flag as well.  Shouldn't we make this consistent?